modules/configuring-default-seccomp-profile.adoc

```diff
@@ -70,13 +70,14 @@ metadata:
     openshift.io/scc: restricted-v2 <1>
     seccomp.security.alpha.kubernetes.io/pod: runtime/default <2>
 ---- 
-<1> The `restricted-v2` is set by default on newly created pods in {product-version}. 
-<2> The `restricted-v2` sets the seccompProfile to `runtime/default` by default. 
+<1> The `restricted-v2` SCC is the SCC added by default, if your workload does not have access to a different SCC. 
+<2> Newly created pods in {product-version} will have the seccomp profile configured to `runtime/default` as mandated by the SCC. 
 
-You can enable the default seccomp profile for a pod or container workload that does not have the seccompProfile set by adding `restricted-v2` to the default service account. 
-
-Follow these steps:
+The default SCC for: 
 
+* A 4.11 fresh install is `restricted-v2`, and pods get the default seccomp profile `runtime/default` set.
+* A 4.11 cluster upgraded from {product-title} version 4.10 or below is `restricted`. In this case if you want to have the seccomp profile `runtime/default` set on all pods follow the step below: 
++
 . Add the `restricted-v2` to the ServiceAccount: 
```

Comment:
This is a tricky one, if the pod can be run with both SCCs, since the last ordering is alphabetical order, the workload will still get assigned to the `restricted` SCC. In that case user will need to force the use of the seccomp profile in the securityContext of the pod or container, which will force the pod to only be admitted by `restricted-v2` (since restricted doesn't allow to configure a seccomp profile).